### ISSUE TITLE
ChoiceGroup: Narrator does not pick text when navigating from one option to another which has Text and icon on Edge

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5284]
If the choicegroup control has text and an icon on it and if one navigates from one option to another Narrator does not announce the new option chosen. This can be reproduced on the fabric react page with Edge and Narrator ON.

**Comment by dzearing** [comment]
@kkjeer Can you please help investigate this? I'm unsure if this is a real issue or an Edge/Navigator issue.

**Comment by JasonGore** [comment]
`ChoiceGroups with images` example (as well as all of the others) is vocalizing options when selected and changed in Edge and Narrator.

@ssingh28 Please confirm you don't have Narrator's [scan mode](https://support.microsoft.com/en-us/help/22809/windows-10-narrator-using-scan-mode) enabled. This seems to be a potential common cause of the types of issues you're reporting.

**Comment by JasonGore** [comment]
Closing as there is no repro with Edge and Narrator (with scan mode disabled) on example page.
